html/templates/header.html

```diff
@@ -23,27 +23,26 @@
 <table align='center' class="top">
 	<tr>
 		<td valign='top'>
-			<div align='center'><font size='3'><b>Baystation 12</b></font></div>
+			<div align='center'><font size='3'><b>Bastion of Hestia</b></font></div>
 			<div align='center'><font size='3'><b><i>A Space Station 13 Project</i></b></font></div>
 			
-			<p><div align='center'><font size='3'><a href="http://wiki.baystation12.net/">Wiki</a> | <a href="https://github.com/Baystation12/Baystation12/">Source code</a></font></div></p>
+			<p><div align='center'><font size='3'><a href="https://github.com/BoHBranch/BoH-Bay">Source code</a></font></div></p>

```

Comment:
```suggestion
			<p><div align='center'><font size='3'><a href="http://wiki.baystation12.net/">Wiki</a> | <a href="https://github.com/BoHBranch/BoH-Bay">Source code</a></font></div></p> //Change this once a proper wiki is up.
```
Until we have a wiki that has the information and shit, you don't want to get rid of such in my opinion.

Reply:
I was told the Discord would suffice, but I kind of agree.

Reply:
Discord means people would be flooding with HOW TO SURGERY and other problems there are answers already written for.